plugins/jcache/src/main/java/org/radargun/service/JCacheLifecycle.java

```diff
@@ -0,0 +1,69 @@
+package org.radargun.service;
+
+import org.radargun.logging.Log;
+import org.radargun.logging.LogFactory;
+import org.radargun.traits.Lifecycle;
+import org.radargun.utils.Utils;
+
+import javax.cache.CacheManager;
+import javax.cache.Caching;
+import javax.cache.configuration.MutableConfiguration;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Properties;
+
+/**
+ * @author Matej Cimbora
+ */
+public class JCacheLifecycle implements Lifecycle {
+
+   private static final Log log = LogFactory.getLog(JCacheLifecycle.class);
+   private JCacheService service;
+   private CacheManager cacheManager;
+
+   public JCacheLifecycle(JCacheService service) {
+      this.service = service;
+   }
+
+   @Override
+   public void start() {
+      URI configFileUri = null;
+      try {
+         if (service.configFile != null) {
+            URL configFile = JCacheLifecycle.class.getClassLoader().getResource(service.configFile);
```

Comment:
I don't think that the URI should point to any configuration file - that's what properties are for. URI should identify the cache manager itself.